test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -182,6 +182,16 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
         $result | Should -Be 1
     }
 
+    It "Test that hex named command executed without tabcompleted prefix will run" {
+        Set-Location $TestDrive
+        New-Item -ItemType Directory -Path $TestDrive -Name "testfolder"
+        $ScriptFolder = Join-Path $TestDrive -ChildPath "testfolder"
+        New-Item -ItemType File -Name 0x.ps1 -Path ".\testfolder\" -Value {Write-Output "Hello"}
```

Comment:
I think you'll need a Push-Location to this folder the script is in for this to work 🙂 

Though tbh the folder here may not be strictly necessary since you're already in TestDrive anyway, maybe just drop the folder creation and `-Path` arg here instead?

Reply:
Why do we need to use `Push-Location` / change the directory structure for this? 
Unfortunately I'm unable to test locally on linux due to work laptop restrictions, so I just verified it ran on Windows and hoped it would work the same on Mac and Linux 😅 
![image](https://user-images.githubusercontent.com/19718175/197311333-44277610-8e42-47ac-96a2-8ed411b3dbb3.png)

Reply:
I changed it a bit but the tests are still failing 😭 . Does Powershell on Linux/Mac work differently in regards to not requiring a prefix for scripts in directories that come under PATH?

Reply:
Oh, right, missed that you were adding it to PATH. No, as far as I'm aware that should be fine... Hmm.